cmd/influx/user_resource_mapping.go

```diff
@@ -3,29 +3,96 @@ package main
 import (
 	"context"
 	"fmt"
+	"os"
 
 	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/cmd/influx/internal"
 )
 
-func membersAddF(urm influxdb.UserResourceMapping, flgs Flags) error {
+func membersListF(ctx context.Context, filter influxdb.UserResourceMappingFilter, flgs Flags) error {
+	mappingSvc, err := newUserResourceMappingService(flgs)
+	if err != nil {
+		return fmt.Errorf("failed to initialize members service client: %v", err)
+	}
+	mps, _, err := mappingSvc.FindUserResourceMappings(ctx, filter)
+	if err != nil {
+		return fmt.Errorf("failed to find members: %v", err)
+	}
+	userSVC, err := newUserService(flgs)
+	if err != nil {
+		return fmt.Errorf("failed to initialize users service client: %v", err)
+	}
+	urs := make([]*influxdb.User, len(mps))
+	ursC := make(chan struct {
+		User  *influxdb.User
+		Index int
+	})
+	errC := make(chan error)
+	for k, v := range mps {
+		go func(k int, v *influxdb.UserResourceMapping) {
```

Comment:
thsi could create a lot of go routines if the number of `mps` is big, perhaps the semaphore here to govern it?  Something like the following would do the trick to limit the number of outgoing requests

```go
sem := make(chan struct{}, maxActive)
for k, v := range mps {
       sem <- struct{}{}
	go func(k int, v *influxdb.UserResourceMapping) {
             defer func() { <-sem }()
             ... other stuffs
```